### ISSUE TITLE
KumuluzEE JWT example returns "Unauthorized"

### ISSUE THREAD

**Issue by Karm** [eclipse/microprofile-starter#215]
KumuluzEE gives:
```
   Claim value within JWT of 'custom-value' :
HTTP ERROR 401

Problem accessing /data/protected. Reason:

    Unauthorized

Powered by Jetty:// 9.4.15.v20190215
```
for JWT example. 

I would like to kindly ask someone familiar with KumuluzEE to download the project:

```
curl -O -J https://start.microprofile.io/api/project?supportedServer=KUMULUZEE&selectedSpecs=JWT_AUTH&selectedSpecs=REST_CLIENT&artifactId=kumuluzee
```
and tell me how to fix it.

Perhaps @MBJuric or @jmezna ?

**Comment by MBJuric** [comment]
We will have a look and will let you know hoe to fix the problem. Thanks.

**Comment by jmezna** [comment]
The _401 Unauthorized_ is caused by a failed JWT token verification, which is a result of a misconfigured key pair.

**Comment by Karm** [comment]
Fixed by #214

**Comment by Karm** [comment]
@jmezna THX. It was really just a stupid error of inconsistent key being pushed into config. Solved.
